components/hero/hero.tsx

```diff
@@ -57,6 +58,10 @@ export const Hero = ({ mainText, aboutText, image, isDarker, children }: IHero)
     ? `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${image.src})`
     : `url(${image.src})}`;
 
+  const theme = useTheme();
+
+  const isShorterLine = useMediaQuery(theme.breakpoints.down("xl"));
```

Comment:
```suggestion
  const isShorterLine = useMediaQuery(theme.breakpoints.down(SCREEN_SIZES.XL));
```

lets avoid magic strings